packages/components/src/components/stepper-horizontal/stepper-horizontal-item/stepper-horizontal-item-styles.ts

```diff
@@ -74,96 +71,99 @@ export const getComponentCss = (state: StepperState, disabled: boolean, theme: T
             (result, _, i) => ({
               ...result,
               [`&(:nth-of-type(${i + 1})) $button::after`]: {
-                backgroundPositionX: `${-i * spriteStepSize}em`,
+                backgroundImage: getInlineSVGBackgroundImage(
+                  getSVGPath(
+                    i + 1,
+                    {
+                      primaryColor,
+                      invertedBaseColor: getInvertedThemedColors(theme).primaryColor,
+                      disabledColor,
+                    },
+                    isStateCurrent
+                  )
+                ),
               },
             }),
             {} as JssStyle
           )),
         ...addImportantToEachRule({
           fontSize: 'inherit',
+          transform: 'translate3d(0,0,0)', // creates new stacking context
```

Comment:
Why would we need one?

Reply:
Frosted glass effect covers the label, therefore the fix with zIndex and a new stacking context